internal/config/config.go

```diff
@@ -118,6 +118,10 @@ type RedactedProfile struct {
 func (p RedactedProfile) MarshalJSON() ([]byte, error) {
 	type profile RedactedProfile
 	pc := profile(p)
+	if pc.Session == "" {
+		pc.Session = "NONE"
+		return json.Marshal(&pc)
+	}
```

Comment:
nit: just to avoid duplication

```suggestion
        session := "NONE"
	if pc.Session != "" {
		session = "REDACTED"
	}
	pc.Session = session
```

Reply:
Thanks -- will update :)